website/client/components/tasks/spells.vue

```diff
@@ -324,16 +329,18 @@ export default {
       }
 
       this.markdown(msg); // @TODO: mardown directive?
+      // @TODO: wrap the quest code below in an `if` for only the two boss attack skills? Collection quests aren't directly affected by skills.
+      // I've commented it out to stop console errors (also `let beforeQuestProgress` above). -- Alys
```

Comment:
Brutal Smash and Burst of Flames increase boss damage (i.e., increase user.party.quest.progress.up). None of the other skills directly change user.party.quest.